src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Factories/MetaData/ApprenticeshipFundingMetaDataFactory.cs

```diff
@@ -23,8 +23,8 @@ public object Create(IReadOnlyList<string> values)
                 PwayCode = values[3].SafeParseInt(),
                 EffectiveFrom = values[5].SafeParseDate(),
                 EffectiveTo = values[6].SafeParseDate(),
-                ReservedValue1 = Convert.ToInt32(values[10].SafeParseDouble()),
-                MaxEmployerLevyCap = Convert.ToInt32(values[12].SafeParseDouble())
+                ReservedValue1 = Convert.ToInt32(values[13].SafeParseDouble()),
+                MaxEmployerLevyCap = Convert.ToInt32(values[16].SafeParseDouble())
```

Comment:
is there any way to use the column header instead of the column number?

Reply:
they have also changed the column header, so use that header wouldn't fix the issue